direct/src/gui/DirectGuiGlobals.py

```diff
@@ -69,6 +69,8 @@
 B1RELEASE = PGButton.getReleasePrefix() + MouseButton.one().getName() + '-'
 B2RELEASE = PGButton.getReleasePrefix() + MouseButton.two().getName() + '-'
 B3RELEASE = PGButton.getReleasePrefix() + MouseButton.three().getName() + '-'
+MWUP = PGButton.getPressPrefix() + MouseButton.wheel_up().getName() + '-'
+MWDOWN = PGButton.getPressPrefix() + MouseButton.wheel_down().getName() + '-'
```

Comment:
I think this might be too obscure of a name.  How about `WHEELUP`?

Reply:
Sure, will add unit tests for it. And yeah, WHEELUP and WHEELDOWN are fine too. I just tried to keep it similar to the other event names in the first place.